tests/ApiCore/Tests/Unit/PathTemplateTest.php

```diff
@@ -156,49 +148,47 @@ public function testMatchEscapedChars()
         );
     }
 
+    /**
+     * @expectedException \Google\ApiCore\ValidationException
+     */
     public function testMatchWildcardWithColonInMiddle()
```

Comment:
For tests which are expected to fail could we follow the naming others tests use? e.g. `testFailUnexpectedEof`

Reply:
Fixed in https://github.com/googleapis/gax-php/pull/209